ovos_plugin_manager/tts.py

```diff
@@ -71,8 +78,10 @@ def create(config=None):
         }
         """
         tts_config = get_tts_config(config)
-        tts_lang = tts_config["lang"]
+        tts_lang = tts_config.get("lang", "en-us")
         tts_module = tts_config.get('module', 'mimic')
+        tts_module = OVOSTTSFactory.get_equivalent_plugin(tts_module) or tts_module
```

Comment:
This effectively blacklists some tts module names; should we see if the module exists as a plugin and fall back to the `OVOSTTSFactory.MAPPINGS`?

Reply:
good call :+1: